src/components/SaveSession.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <v-card>
+    <v-card-title>
+      Saving Session State
+      <v-spacer />
+      <v-btn icon @click="$emit('close')"><v-icon>mdi-close</v-icon></v-btn>
+    </v-card-title>
+    <v-card-text>
+      <v-text-field
+        v-on:keyup.enter="saveSession"
+        v-model="fileName"
+        hint="The filename to use for the session state file."
+        label="Session State Filename"
+      />
+    </v-card-text>
+    <v-card-actions>
+      <v-spacer></v-spacer>
+      <v-btn color="secondary" @click="saveSession">
+        <v-icon class="mr-2">mdi-content-save-all</v-icon>
+        <span>Save</span>
+      </v-btn>
+    </v-card-actions>
+  </v-card>
+</template>
+
+<script lang="ts">
+import { defineComponent, ref } from '@vue/composition-api';
+import { save } from '../io/state-file';
+
+const DEFAULT_FILENAME = 'volview.zip';
```

Comment:
Maybe `session.volview.zip`. We don't need to hide it being a zip file, but keeping a sub-extension of `volview` is helpful.

Reply:
Do we want to enforce the sub-extension?